src/v/cluster/controller.cc

```diff
@@ -98,12 +98,8 @@ ss::future<> controller::wire_up() {
       .then([this] { _probe.start(); });
```

Comment:
> The rest of the subsystems will have a dependency on cluster discovery,
which, once automatic node ID assignment is supported, will be what
determines the node ID.

neat!